mjx/mujoco/mjx/_src/sensor.py

```diff
@@ -600,3 +602,79 @@ def _framelinacc(cvel, cacc, offset):
   )
 
   return d.replace(sensordata=sensordata)
+
+
+def energy_pos(m: Model, d: Data) -> Data:
+    """Calculates position-dependent energy (potential).
+    """
+        
+    # Initialize potential energy
+    energy = jp.array(0.0)
```

Comment:
mega nitpick: please move this variable definition to right after the if statement guard